### PR TITLE
User can request items

### PR DESCRIPTION
**Pull request by kaylawoodbury**
### User Story
```
As a resident
In order to request items from someone shopping
I would like to contact and request items from that user
```
## Changes proposed in this pull request:
*  Adds request pong button
* Add request pong form and functionality
* Adds styling
* Updates previous tests/code to incorporate changes for requests

## What We have learned working on this feature:
* Hiding items can be tricky
* Error statuses do not send messages from the backend
* To know when to stop

## Screenshots (Client)

Screenshot 2020-04-16 at 13 33 49
Screenshot 2020-04-16 at 13 34 11
Screenshot 2020-04-16 at 13 39 32

## Lo-Fi's (Client)
image (3)
image (4)

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -7,9 +7,11 @@ import axios from "axios";
 import TripsList from "./screens/TripsList";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
+import RequestForm from "./screens/RequestForm";
 
 axios.defaults.baseURL = "https://co-ping.herokuapp.com";
 
+
```
Comment (emtalen) [inline App.js]: remove extra line

**Review by emtalen — COMMENTED** [review]
Awesome Possum!! grr 🙌

File: cypress/integration/visitorCanCreateNewRequest.feature.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    cy.visit("/");
+    cy.window().then((window) => {
+      window.store.dispatch({
+        type: "AUTHENTICATE",
+        payload: { authenticated: true, userId: 2 },
+      });
+    });
+    cy.get("#trip-list-button").click();
+  });
+  it("send an empty request three items", () => {
```
Comment (johanbounce) [inline cypress/integration/visitorCanCreateNewRequest.feature.js:62]: I would love a linebreak before the new it-block

File: screens/HomeScreen.jsx [hunk 3]
```diff
@@ -26,8 +27,8 @@ const HomeScreen = ({ navigation }) => {
       >
         <Image style={styles.logo} source={Logo} alt="Co-Ping logo" />
```
Comment (johanbounce) [inline screens/HomeScreen.jsx]: Line

File: screens/HomeScreen.jsx [hunk 4]
```diff
@@ -26,8 +27,8 @@ const HomeScreen = ({ navigation }) => {
...
 
-        <Text style={styles.logoutMessage}>
-          {!authenticated && logoutMessage}
+        <Text style={styles.authMessage} nativeID="auth-message">
+          {authenticated ? loginMessage : logoutMessage}
         </Text>
```
Comment (johanbounce) [inline screens/HomeScreen.jsx]: line

**Review by johanbounce — COMMENTED** [review]
will approve  when fixed

**Review by pgaunitz — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (kaylawoodbury) [reply App.js]: removed

**Review by johanbounce — APPROVED** [review]

File: cypress/integration/visitorCanCreateNewRequest.feature.js (hunk 2 above)
Comment (kaylawoodbury) [reply cypress/integration/visitorCanCreateNewRequest.feature.js:62]: added

File: screens/HomeScreen.jsx (hunk 3 above)
Comment (kaylawoodbury) [reply screens/HomeScreen.jsx]: removed

File: screens/HomeScreen.jsx (hunk 4 above)
Comment (kaylawoodbury) [reply screens/HomeScreen.jsx]: removed

**Review by johanbounce — APPROVED** [review]

**Comment by johanbounce** [comment]
NICE JOB GUYS!!!!

**Pull request merged**